builtin/mainmenu/settings/dlg_settings.lua

```diff
@@ -0,0 +1,514 @@
+--Minetest
+--Copyright (C) 2022 rubenwardy
+--
+--This program is free software; you can redistribute it and/or modify
+--it under the terms of the GNU Lesser General Public License as published by
+--the Free Software Foundation; either version 2.1 of the License, or
+--(at your option) any later version.
+--
+--This program is distributed in the hope that it will be useful,
+--but WITHOUT ANY WARRANTY; without even the implied warranty of
+--MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--GNU Lesser General Public License for more details.
+--
+--You should have received a copy of the GNU Lesser General Public License along
+--with this program; if not, write to the Free Software Foundation, Inc.,
+--51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+
+local component_funcs =  dofile(core.get_mainmenu_path() .. DIR_DELIM ..
+		"settings" .. DIR_DELIM .. "components.lua")
+
+local quick_shader_component =  dofile(core.get_mainmenu_path() .. DIR_DELIM ..
+		"settings" .. DIR_DELIM .. "shader_component.lua")
+
+
+local full_settings = settingtypes.parse_config_file(false, true)
+local info_icon_path = defaulttexturedir .. "settings_info.png"
+local reset_icon_path = defaulttexturedir .. "settings_reset.png"
```

Comment:
```suggestion
local reset_icon_path = core.formspec_escape(defaulttexturedir .. "settings_reset.png")
```
And the same with `info_icon_path`.
Otherwise the icons break if Minetest is located somewhere like `C:\Users\Blah\Downloads\msvc-x64-portable\minetest-5.8.0-dev-win64\bin`.
(Not sure if this is the case because of those Windows backslashes, or something else.)

**Before:**
![image](https://user-images.githubusercontent.com/5585984/234787497-b71083b2-9740-4bc2-ba02-cc811c9ab974.png)

**After:**
![image](https://user-images.githubusercontent.com/5585984/234789064-40273690-faaa-430b-a6b5-903ff39311f3.png)